src/commands/deployment/correlate.ts

```diff
@@ -0,0 +1,169 @@
+import chalk from 'chalk'
+import {Command, Option} from 'clipanion'
+import simpleGit from 'simple-git'
+
+import {getCISpanTags} from '../../helpers/ci'
+import {gitRepositoryURL, gitLocalCommitShas, gitCurrentBranch} from '../../helpers/git/get-git-data'
+import {Logger, LogLevel} from '../../helpers/logger'
+import {retryRequest} from '../../helpers/retry'
+import {CI_PROVIDER_NAME, CI_ENV_VARS, GIT_REPOSITORY_URL, GIT_SHA} from '../../helpers/tags'
+import {getApiHostForSite, getRequestBuilder} from '../../helpers/utils'
+
+/**
+ * This command collects environment variables and git information to correlate commits from the
+ * source code repository to the configuration repo repository. This allows to connect pipelines triggering
```

Comment:
configuration repo repository? Is there any other way to refer to this?

Reply:
was a typo, nice catch